gst/nnstreamer/tensor_common.c

```diff
@@ -183,6 +184,10 @@ gst_tensor_info_is_equal (const GstTensorInfo * i1, const GstTensorInfo * i2)
     return FALSE;
   }
 
+  if (i1->layout != i2->layout) {
```

Comment:
Compare the layouts :
This will make caps-negotiation failure when incoming meta (usually non-layout) is different with the properties in tensor-filter.

Reply:
How about if any of the layout is `none`, then comparison is not performed?

Reply:
As discussed with @myungjoo , there will be 4 layouts, `ANY`, `NONE`, `NHWC` and `NCHW`. `ANY` can match with all, however others are specific. I will update this PR as per this.